app/models/teammate.rb

```diff
@@ -45,6 +45,10 @@ def name
     user ? user.name : ""
   end
 
+  def ratings_count
+    user.ratings.count
```

Comment:
I think this would work better if scoped by event, so you don't get ratings from other events showing up.